framework/base/ErrorHandler.php

```diff
@@ -262,7 +275,12 @@ public function handleError($code, $message, $file, $line)
      */
     public function handleFatalError()
     {
-        unset($this->_memoryReserve);
+        $this->_memoryReserve = null;
```

Comment:
As far as I remember it works differently memory-wise. Needs to be re-checked.

Reply:
@samdark i'm think is more correctly way

Reply:
Not in this case. Setting null does not remove variable from the symbol table thus freeing a bit less memory. Let's revert to `unset()`. See https://stackoverflow.com/questions/584960/whats-better-at-freeing-memory-with-php-unset-or-var-null

Reply:
@samdark it's not variable, it's property

Reply:
Yes. That's the goal. Unset will remove it from symbol table freeing up a bit more memory.

Reply:
There will be no second access to this property.

Reply:
@samdark economy 30-40 bytes? no reasons to use ugly solutions like this - yii have many unoptimized places